VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/context-menu-header/context-menu-header.component.html

```diff
@@ -0,0 +1 @@
+  <div class="govuk-heading-s horizontal-spacer" id="context-menu-header">{{(isPrivateConsultation ? 'private-consultation-room-controls.consultation-controls' : 'private-consultation-room-controls.hearing-controls') | translate}}</div>
```

Comment:
Any reason to make this a separate component? It's just 1 element.

Reply:
I see the comment regarding this now. I think it's fine.

Reply:
As I have mentioned in this pr, I had difficulties mocking video service in the private-consultation-room-controls component and found it easier to test it creating a separate component this way. I also didn't see this would create any drawbacks as the files will get eventually minified. However if this isnt ideal I would need some help with refactoring the private-consultation-room-controls test file as its currently not using a test bed and has a lot of dependencies.